### ISSUE TITLE
Error messages incorrect formatting, fails on some basic classes. 

### ISSUE THREAD

**Issue by douglasb99** [gregjacobs/js-to-ts-converter#9]
/home/douglas/.nvm/versions/node/v12.11.1/lib/node_modules/js-to-ts-converter/node_modules/ts-morph/dist/manipulation/helpers/getRangeWithoutCommentsFromArray.js:31
        throw new errors.NotImplementedError(`Unexpected! Inserted ${length} child/children, but ${children.length} were inserted.`);
 

All errors seem to print like this ie without doing the string interpolation being performed. 

Also it fails on some simple files such as a dir containing  this file (the file it fails on) 

export class Base extends Error {
    constructor(message = '') {
        super(message)
        this.name = this.constructor.name

        Error.captureStackTrace && Error.captureStackTrace(this, this.constructor)
    }
}

BTW It would be far better if it had a flag to convert all functions to typescript funcs with "any" types to reduce manual labor during migration or better  even be able to be used in conjunction with a typescript declaration file so we could build a typescript code base automatically taking types from that (many projects have one written by hand).

**Comment by douglasb99** [comment]
Furthermore I don't know exactly what it does but if it holds state between files and directories would be good to have an exclude option, to exclude certain files/sub-directories if it does not have.

**Comment by XLCYun** [comment]
I ran into a similar problem while converting a Class extended from Error. This might be what happened: 
It seems that `js-to-ts-converter` had treated `constructor` as a property of class, it makes sense that in js we declare and add property by simply set a value to `this.xxx` in the `constructor` function,  so parse the code of `constructor` function, and find any accessing like `this.xxx`, we get the properties of the class, but when we have a class extends from Error, in constructor, we usually need to access this.constructor(pass as argument to `Error.captureStackTree`), and `js-to-ts-converter` mistakes `constructor` is a property of this class. Here is the output error message: 
```
Adding property declarations for properties: 'name', 'code', 'userMessage', 'constructor'
C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\node_modules\ts-morph\dist\manipulation\helpers\getRangeWithoutCommentsFromArray.js:18
                throw new errors.NotImplementedError(`Unexpected! Inserting syntax kind of ${utils_1.getSyntaxKindName(expectedKind)}` +
                ^
NotImplementedError: Unexpected! Inserting syntax kind of PropertyDeclaration, but Constructor was inserted.
    at Object.getRangeWithoutCommentsFromArray (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\node_modules\ts-morph\dist\manipulation\helpers\getRangeWithoutCommentsFromArray.js:18:23)
    at Object.insertIntoBracesOrSourceFileWithGetChildren (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\node_modules\ts-morph\dist\manipulation\manipulations\insertion.js:183:22)
    at insertChildren (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\node_modules\ts-morph\dist\compiler\ast\class\base\ClassLikeDeclarationBase.js:428:27)
    at ClassDeclaration.insertProperties (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\node_modules\ts-morph\dist\compiler\ast\class\base\ClassLikeDeclarationBase.js:169:20)
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\dist\converter\add-class-property-declarations\add-class-property-declarations.js:63:26
    at Array.forEach (<anonymous>)
    at Object.addClassPropertyDeclarations (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\dist\converter\add-class-property-declarations\add-class-property-declarations.js:37:34)
    at Object.convert (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\dist\converter\convert.js:26:54)
    at doConvert (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\dist\js-to-ts-converter.js:62:22)
    at Object.convertJsToTsSync (C:\Users\Administrator\AppData\Roaming\npm\node_modules\js-to-ts-converter\dist\js-to-ts-converter.js:40:35)
```
Highlights:
```
verbose:            Adding property declarations for properties: 'name', 'code', 'userMessage', 'constructor'
```
It tried to add a property declarations for 'constructor'.
```
NotImplementedError: Unexpected! Inserting syntax kind of PropertyDeclaration, but Constructor was inserted.
```

**Comment by creepinson** [comment]
I have the exact same problem as seen above.

**Comment by agustinvinao** [comment]
Anyone found a workaround or a solution?

**Comment by XLCYun** [comment]
I have only one file one class that use `this.constructor` in the constructor function, so I just simply convert it by hand. 

> Anyone found a workaround or a solution?

**Comment by gregjacobs** [comment]
Fixed in 0.17.1
